src/handlers/transfers/handler.js

```diff
@@ -269,106 +226,144 @@ const fulfil = async (error, messages) => {
       histTimerEnd({ success: false, fspId: Config.INSTRUMENTATION_METRICS_LABELS.fspId })
       return true
     }
-    const metadata = message.value.metadata
-    const transferId = message.value.id
-    const payload = message.value.content.payload
-    const headers = message.value.content.headers
-    if (metadata.event.type === TransferEventType.FULFIL &&
-      (metadata.event.action === TransferEventAction.COMMIT ||
-        metadata.event.action === TransferEventAction.REJECT ||
-        metadata.event.action === TransferEventAction.ABORT)) {
-      const existingTransfer = await TransferService.getById(transferId)
+    // Action Letter (AL) is used to denote breadcrumb flow end
+    const AL = action === TEA.COMMIT ? 'C' : (action === TEA.REJECT ? 'R' : (action === TEA.ABORT ? 'A' : '?'))
```

Comment:
For readability please rename to actionLetter also i would store 'R', 'A' and '?' under constants with a name that explains what it is or create an enum for all this lettering

Reply:
ok